### ISSUE TITLE
Issue installing KVM guest from FCOS Live ISO (have to add console=ttyS0)

### ISSUE THREAD

**Issue by cgruver** [coreos/fedora-coreos-tracker#597]
I am attempting to install an FCOS guest VM on a KVM hypervisor.

Host OS:  CentOS 8

FCOS: stable - 32.20200715.3.0

The iso has a customized ignition injected with: `coreos-installer iso embed --config...`

The guest is configured as follows:

```
virt-install --name okd4-snc-bootstrap --memory 14336 --vcpus 2 --disk size=100,path=/VirtualMachines/okd4-snc-bootstrap/rootvol,bus=sata --cdrom /tmp/snc-bootstrap.iso --network bridge=br0 --mac=${BOOT_MAC} --graphics none --noautoconsole
```

The guest VM hangs after `Loading /images/initramfs.img...ok`, unless I interrupt the boot and add the karg `console=ttyS0`

**Comment by jlebon** [comment]
Right, this is no different than other install ISOs which by default don't output to the serial console. That said, we are planning to support appending kernel arguments into the ISO, which will help with this. See https://github.com/coreos/coreos-installer/pull/341.
